hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTaskWithLegacy.java

```diff
@@ -136,7 +136,15 @@ private void processWithFileSystemLayout(OmKeyInfo updatedKeyInfo,
                                            OMDBUpdateEvent.OMDBUpdateAction action,
                                            Map<Long, NSSummary> nsSummaryMap)
       throws IOException {
-    setKeyParentID(updatedKeyInfo);
+
+    try {
+      setKeyParentID(updatedKeyInfo);
```

Comment:
Same warn message we are logging in `setKeyParentID` method also. `processWithFileSystemLayout` is the caller of `setKeyParentID`, can we log different message which can mention what that caller is doing.

Reply:
I couldn't find any logging in the `setKeyParentID` method, only the IOException is there.
https://github.com/apache/ozone/blob/00f89e60563a1e2d290cd49f7175dac87d533223/hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/NSSummaryTaskWithLegacy.java#L308-L335